webclient/ez/views.py

```diff
@@ -680,10 +717,18 @@ def post(self, request, *args, **kwargs):
             and eo.archeologicky_zaznam.lokalita.igsn
         ):
             lokalita_update = eo.archeologicky_zaznam.lokalita
-        eo.close_active_transaction_when_finished = True
-        eo.delete()
-        if lokalita_update:
-            lokalita_update.igsn_update()
+        try:
+            if lokalita_update:
+                lokalita_update.igsn_update()
+            eo.close_active_transaction_when_finished = True
+            eo.delete()
```

Comment:
Tady je stále otočené to pořadí... Nejdřív se musí odkaz smazat, pak se teprve může volat igsn_update, jinak volání nic nezmění.

Reply:
Přesunuto.